### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by 6543**
* Resolve https://github.com/woodpecker-ci/woodpecker/issues/468#issuecomment-968364153
* Move info from README into docs
* Fix links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,92 +48,39 @@ Please consider to donate and become a backer. 🙏 [[Become a backer](https://o
...
-    template: config/k8s/service.yml
-```
+[Read More](https://woodpecker-ci.org/docs/usage/pipeline-syntax#steps)
+
+### Plugins
+
+Woodpecker has official plugins https://woodpecker-ci.org/plugins,
+but you can also use own one.
+
+[Read More](https://woodpecker-ci.org/docs/usage/plugins/plugins)
```
Comment (anbraten) [inline README.md:65]: I will try to improve that text a bit

```md
# Woodpecker

TODO

[Learn More](https://woodpecker-ci.org/docs/usage/intro)
```

File: README.md [hunk 2]
```diff
@@ -48,92 +48,39 @@ Please consider to donate and become a backer. 🙏 [[Become a backer](https://o
...
-  deploy-to-k8s:
-    image: laszlocloud/my-k8s-plugin
-    template: config/k8s/service.yml
-```
+[Read More](https://woodpecker-ci.org/docs/usage/pipeline-syntax#steps)
+
+### Plugins
+
+Woodpecker has official plugins https://woodpecker-ci.org/plugins,
+but you can also use own one.
```
Comment (jolheiser) [inline README.md:63]:
```suggestion
but you can also use your own.
```

File: README.md [hunk 3]
```diff
@@ -48,92 +48,39 @@ Please consider to donate and become a backer. 🙏 [[Become a backer](https://o
...
+[Read More](https://woodpecker-ci.org/docs/usage/plugins/plugins)
 
 ## Documentation
 
 https://woodpecker-ci.org/
 
 ## Who uses Woodpecker?
 
-Currently, I know of one organization using Woodpecker. With 50+ users, 130+ repos and more than 1100 builds a week.
+[Codeberg](https://codeberg.org), We ourselve and a lot more.
```
Comment (jolheiser) [inline README.md:73]:
```suggestion
[Codeberg](https://codeberg.org), the woodpecker project itself, and many others not listed.
```

File: docs/docs/20-usage/20-pipeline-syntax.md [hunk 4]
```diff
@@ -101,7 +101,7 @@ pipeline:
 
 If required, Woodpecker can be made to skip whole pipelines based on `when`. This could be utilised to ensure compliance that only certain jobs run on certain agents (regional restrictions). Or targeting architectures.
 
-This is achieved by ensuring the `when` block is on the root level. Rather than 
+This is achieved by ensuring the `when` block is on the root level. Rather than
```
Comment (jolheiser) [inline docs/docs/20-usage/20-pipeline-syntax.md:104]:
```suggestion
This is achieved by ensuring the `when` block is on the root level.
```

**Review by jolheiser — COMMENTED** [review]
Wording nits

File: README.md (hunk 2 above)
Comment (jolheiser) [reply README.md:63]: Also, should `official plugins` just be a markdown link to https://woodpecker-ci.org/plugins ?

File: README.md (hunk 1 above)
Comment (6543) [reply README.md]: @anbraten do you have suggestions?

**Review by jolheiser — APPROVED** [review]
LGTM pending any other feedback from anbraten

File: docs/docs/20-usage/20-pipeline-syntax.md [hunk 5]
```diff
@@ -198,6 +198,24 @@ pipeline:
...
+- Woodpecker clones the source code in the beginning pipeline
+- Changes to files are persisted through steps as the same volume is mounted to all steps
+
+```yaml
+# .woodpecker.yml
+pipeline:
+  build:
+    image: debian
+    commands:
+      - touch myfile
```
Comment (6543) [inline docs/docs/20-usage/20-pipeline-syntax.md:212]:
```suggestion
      - echo "test content" > myfile
```

?

**Comment by 6543** [comment]
ok I'll merge - more can be done in followup pull

**Pull request merged**